### PR TITLE
OM - 598 | Jässäri admin list view

### PR DESCRIPTION
**Pull request by tuovinensanttu**
This PR contains few bug fixes.

- Search in lists will always query data from backend.
- Shows amount of search results below table.
- Added MembershipStatus to list view.
- Added missing localizations in list view.

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -18,8 +18,14 @@
     "birthDate": "Birth date (dd.mm.yyyy)",
     "birthDateWithoutHelp": "Birth date",
     "membershipNumber": "Membership number",
+    "membershipStatus": "Status",
+    "PENDING": "Pending",
+    "ACTIVE": "Active",
+    "EXPIRED": "Expired",
```
Comment (Jokinen) [inline src/i18n/en.json]: These keys are in `CONSTANT_CASE` case when all the other keys are in `camelCase`. Looks like application logic may be seeping into the translations here.

This is just a small FYI. Naming translation keys is something that I don't know how to do well.

File: src/i18n/en.json [hunk 2]
```diff
@@ -18,8 +18,14 @@
...
     "phone": "Phone number",
     "photoUsage": "Photo usage",
+    "approved": "Yes",
```
Comment (Jokinen) [inline src/i18n/en.json]: The translation path does not seem to match with the translation. If you were a translator and had to add a translation for this key, you might be puzzled. The same probably applies to the next dev who is going through these translations--or would like to add a label for "approved".

This quirk seems  to have same motivations as the above one.

Comment (tuovinensanttu) [reply src/i18n/en.json]: Renamed translation key to `photoUsageApproved` to improve matching.

File: src/i18n/en.json (hunk 1 above)
Comment (tuovinensanttu) [reply src/i18n/en.json]: I'm not 100% sure myself. I made a little change and ended up going with same approach which is used with languages.

>   "PROFILE_STATUS":  {
    "PENDING": "Odottaa",
    "ACTIVE": "Aktiivinen",
    "EXPIRED": "Vanhentunut"
  },


>

**Pull request merged**